torchtext/experimental/datasets/sequence_tagging.py

```diff
@@ -0,0 +1,155 @@
+import torch
+
+from torchtext.experimental.datasets import raw
+from torchtext.vocab import build_vocab_from_iterator
+from torchtext.experimental.functional import (
+    vocab_func,
+    totensor,
+    sequential_transforms,
+)
+
+
+def _build_vocab(data):
+    total_columns = len(data[0])
+    data_list = [[] for _ in range(total_columns)]
+    vocabs = []
+
+    for line in data:
+        for idx, col in enumerate(line):
+            data_list[idx].append(col)
+
+    for it in data_list:
+        vocabs.append(build_vocab_from_iterator(it))
+
+    return vocabs
+
+
+def _setup_datasets(dataset_name,
+                    root=".data",
+                    vocabs=None,
+                    data_select=("train", "valid", "test")):
+    train, val, test = DATASETS[dataset_name](root=root)
+    raw_data = {
```

Comment:
you might add some checks here
https://github.com/pytorch/text/blob/2aa8858272d9b46b71194616ab948a087081207f/torchtext/experimental/datasets/text_classification.py#L80-L83

Reply:
done